README.md

```diff
@@ -1780,11 +1780,10 @@ inputs to PDF fields.
 
 #### application.yaml properties
 
-| Name                            | Description                                                                                                                        | Example                                 |
-|---------------------------------|------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------|
-| `pdf`                           | The path to the pdf template file the FFB library should use when generating your PDF. The path should begin with a forward slash. | `pdf: /pdfs/ubi/Multipage-UBI-Form.pdf` |
```

Comment:
oh, this should be "path", not pdf.

Reply:
We actually realized it's not needed at all anymore after a change we made a while back 😅